HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -24,7 +24,7 @@ code::
 Server.default.options.device_("MOTU 828");
 ::
 
-On macOS you can programmatically obtain a list of available audio devices without booting the server:
+You can programmatically obtain a list of available audio devices without booting the server:
```

Comment:
Sounds good. I went with your first suggestion, since it seems more user-friendly. It the user compiles SC with a different backend, they likely understand the implications of different backends.